src/mpriscontroller_p.h

```diff
@@ -0,0 +1,336 @@
+//
+// This file is not part of the public API.  This header file may
+// change from version to version without notice, or even be
+// removed.
+//
+// We mean it.
+//
+//
+
+
+#ifndef MPRISCONTROLLER_P_H
+#define MPRISCONTROLLER_P_H
+
+#include <DBusExtendedAbstractInterface>
+
+#include <QtCore/QObject>
+#include <QtCore/QByteArray>
+#include <QtCore/QList>
+#include <QtCore/QMap>
+#include <QtCore/QString>
+#include <QtCore/QStringList>
+#include <QtCore/QVariant>
+#include <QtDBus/QtDBus>
+
+/*
+ * Proxy class for interface org.mpris.MediaPlayer2
+ */
+class MprisRootInterface: public DBusExtendedAbstractInterface
+{
+    Q_OBJECT
+public:
+    static inline const char *staticInterfaceName()
+    { return "org.mpris.MediaPlayer2"; }
+
+public:
+    MprisRootInterface(const QString &service, const QString &path, const QDBusConnection &connection, QObject *parent = 0);
+
+    ~MprisRootInterface();
+
+    Q_PROPERTY(bool CanQuit READ canQuit NOTIFY canQuitChanged)
+    inline bool canQuit()
+    { return qvariant_cast< bool >(internalPropGet("CanQuit", &m_canQuit)); }
+
+    Q_PROPERTY(bool CanRaise READ canRaise NOTIFY canRaiseChanged)
+    inline bool canRaise()
+    { return qvariant_cast< bool >(internalPropGet("CanRaise", &m_canRaise)); }
+
+    Q_PROPERTY(bool CanSetFullscreen READ canSetFullscreen NOTIFY canSetFullscreenChanged)
+    inline bool canSetFullscreen()
+    { return qvariant_cast< bool >(internalPropGet("CanSetFullscreen", &m_canSetFullscreen)); }
+
+    Q_PROPERTY(QString DesktopEntry READ desktopEntry NOTIFY desktopEntryChanged)
+    inline QString desktopEntry()
+    { return qvariant_cast< QString >(internalPropGet("DesktopEntry", &m_desktopEntry)); }
+
+    Q_PROPERTY(bool Fullscreen READ fullscreen WRITE setFullscreen NOTIFY fullscreenChanged)
+    inline bool fullscreen()
+    { return qvariant_cast< bool >(internalPropGet("Fullscreen", &m_fullscreen)); }
+    inline void setFullscreen(bool value)
+    { m_fullscreen = value; internalPropSet("Fullscreen", QVariant::fromValue(value), &m_fullscreen); }
+
+    Q_PROPERTY(bool HasTrackList READ hasTrackList NOTIFY hasTrackListChanged)
+    inline bool hasTrackList()
+    { return qvariant_cast< bool >(internalPropGet("HasTrackList", &m_hasTrackList)); }
+
+    Q_PROPERTY(QString Identity READ identity NOTIFY identityChanged)
+    inline QString identity()
+    { return qvariant_cast< QString >(internalPropGet("Identity", &m_identity)); }
+
+    Q_PROPERTY(QStringList SupportedMimeTypes READ supportedMimeTypes NOTIFY supportedMimeTypesChanged)
+    inline QStringList supportedMimeTypes()
+    { return qvariant_cast< QStringList >(internalPropGet("SupportedMimeTypes", &m_supportedMimeTypes)); }
+
+    Q_PROPERTY(QStringList SupportedUriSchemes READ supportedUriSchemes NOTIFY supportedUriSchemesChanged)
+    inline QStringList supportedUriSchemes()
+    { return qvariant_cast< QStringList >(internalPropGet("SupportedUriSchemes", &m_supportedUriSchemes)); }
```

Comment:
This lines are autogenerated with qdbusxml2cpp, copied together here and modified.

I would rather not modify them since I expect I could replace this in the future when qtdbusextended would be providing its own modified qdbusextendedxml2cpp tool.